services/mediapackagevod/src/main/resources/codegen-resources/customization.config

```diff
@@ -0,0 +1,6 @@
+{
+  "renameShapes": {
+    // Do not keep adding to this list. Require the service team to name enums like they're naming their shapes.
+    "__PeriodTriggersElement": "PeriodTriggersElement"
```

Comment:
Is this breaking change?

Reply:
No, the existing __s were being stripped. My change no longer strips, so I did the rename manually.